app/views/admin/surveys/show.html.erb

```diff
@@ -38,8 +38,14 @@ as well as a link to its edit page.
       <i class="fas fa-tachometer-alt mr-1"></i> <%= t('dashboard') %>
     <% end %>
 
-    <%= link_to(survey_entries_path(page.resource), class: 'button') do %>
-      <i class="fas fa-eye mr-1"></i> <%= t('administrate.surveys.show.preview') %>
```

Comment:
I’d rather not remove the explanation complately but add a tooltip. There is already one on the checkmark when you completed a survey, you could copy code from there